docs/source/remote-data-services.rst

```diff
@@ -61,16 +71,22 @@ providing ``storage_options``, a dictionary of parameters to pass on. Examples:
    df = dd.read_parquet('s3://bucket/path',
                         storage_options={'anon': True, 'use_ssl': False})
 
-Further details on how to provide configuration for each back-end
-is listed next.
+Further details on how to provide configuration for the main back-ends
+is listed next, but further details can be found in the docs pages of the
+relavant back-end.
 
 Each back-end has additional installation requirements and may not be available
-at runtime. The dictionary ``dask.bytes.core._filesystems`` contains the
-currently available file systems. Some require appropriate imports before use.
+at runtime. The dictionary ``fsspec.registry`` contains the
+currently imported file systems, and to see which backends ``fsspec`` knows how
+to import, you can do
 
-The following list gives the protocol shorthands and the back-ends to which
-they refer.
+.. code-block:: python
 
+    from fsspec.registry import known_implementations
+    known_implementations
+
+Note that some backends appear twice, if they can be referenced with multiple
+protocol strings, e.g., "http" and "https".
```

Comment:
Copy-editing people often yell at me for using "e.g.", which is apparently unclear for non-native english speakers

```suggestion
protocol strings, like "http" and "https".
```

Honestly, even as a native english speaker I'm not entirely sure what it means :)